tests/queries/0_stateless/01658_read_file_to_stringcolumn.sh

```diff
@@ -9,16 +9,12 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 
 # Data preparation.
 
-# Now we can get the user_files_path by use the table file function for trick. also we can get it by query as:
-#  "insert into function file('exist.txt', 'CSV', 'val1 char') values ('aaaa'); select _path from file('exist.txt', 'CSV', 'val1 char')"
-CLICKHOUSE_USER_FILES_PATH=$(clickhouse-client --query "select _path, _file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception | awk '{gsub("/nonexist.txt","",$9); print $9}')
-
-mkdir -p ${CLICKHOUSE_USER_FILES_PATH}/
-echo -n aaaaaaaaa > ${CLICKHOUSE_USER_FILES_PATH}/a.txt
-echo -n bbbbbbbbb > ${CLICKHOUSE_USER_FILES_PATH}/b.txt
-echo -n ccccccccc > ${CLICKHOUSE_USER_FILES_PATH}/c.txt
+mkdir -p ${USER_FILES_PATH}/
```

Comment:
Is this really necessary? I'd expect this to exist. Maybe it was meant to be a subdirectory?